qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -66,6 +66,8 @@ public enum QuteErrorCode implements IQuteErrorCode {
 	UnexpectedMemberTypeInCaseSection("Unexpected type `{0}` in `{1}`. Expected `{2}`."),
 	UnexpectedValueInCaseSection("Unexpected value `{0}` in `{1}`. Expected value of type `{2}`."),
 	InvalidParentInCaseSection("`{0}` section must be hosted in a #switch or #when section."),
+	UnexpectedParameterInCaseSection("Unexpected operand `{0}`. The operator `{1}` in the `#{2}` section expects only one parameter."),

```

Comment:
I would like to avoid using Case in error code when it is generic, because we could use this error code for other section: 

 * CaseParameterRequired -> SectionParameterRequired
 * UnexpectedParameterInCaseSection -> what about UnexpectedOperand ?

Reply:
CaseParameterRequired => MissingParameter
UnexpectedParameterInCaseSection => UnexpectedParameter